src/atomate2/common/jobs.py

```diff
@@ -6,25 +6,46 @@
 
 from atomate2 import SETTINGS
 
+__all__ = ["structure_to_primitive", "structure_to_conventional"]
+
 
 @job
-def symmetrize_structure(
-    structure: Structure, symprec: float = SETTINGS.SYMPREC
-) -> Structure:
+def structure_to_primitive(structure: Structure, symprec: float = SETTINGS.SYMPREC):
     """
-    Symmetrize a structure.
+    Job that creates a standard primitive structure.
 
     Parameters
     ----------
-    structure : .Structure
-        A structure.
-    symprec : float
-        The symmetry precision.
+        structure: Structure object
```

Comment:
I don't think these should be indented an extra level.

Reply:
Corrected.

Reply:
And, tests have to be restarted, of course. Sorry.